Kickstarter-iOS/Views/Controllers/CheckoutNavigationController.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+final class CheckoutNavigationController: UINavigationController {
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.delegate = self
```

Comment:
Just to separate call to super and make it stand out a bit more. Definitely can live without this :) 

```suggestion

    self.delegate = self
```